src/routes/redirector.ts

```diff
@@ -9,10 +9,21 @@ export default async function (fastify: FastifyInstance): Promise<void> {
     '/:postId',
     async (req, res) => {
       const con = await createOrGetConnection();
-      const post = await con.getRepository(ArticlePost).findOne({
-        select: ['id', 'url', 'tagsStr'],
-        where: [{ id: req.params.postId }, { shortId: req.params.postId }],
-      });
+      const q = con
+        .createQueryBuilder()
+        .select([
+          'post.id AS id',
+          'post.url AS url',
+          'post.tagsStr AS "tagsStr"',
```

Comment:
I believe we don't need the `AS` part and also the `post.` prefix

Reply:
With out `AS` it returns them as `post_id` etc